### DISCUSSION TITLE
how to use <|startoflm|> in whisper

### DISCUSSION THREAD

**Discussion by mynet1314** [openai/whisper#414] [Q&A] [answered]
I want to know how to use <|startoflm|> which is one of 51685 tokens, I don't know how to use it. 

Could anyone give some tips about it?

**Comment by jongwook** [comment, marked as answer]
We experimented with jointly training the decoder to work as a language model, and `<|startoflm|>` served as a start-of-text token for those training examples (where it'd skip the cross-attention and work like a GPT-2), but we ended up not using the setup after seeing no significant improvement over just using the ASR data. So all released models have never seen this token. In hindsight after seeing many spelling errors (e.g. #266), I think this could've been still helpful for multilingual training.

**Comment by mynet1314** [comment]
Thank you for your reply. Understood.
